packages/cml/src/getRgxInstance/getRgxInstance.ts

```diff
@@ -1,2 +1,5 @@
 export const getRgxInstance = () =>
-  new RegExp('{{([^:^{]*):(\\w+)(?:\\((.+)?\\))?(\\[([^\\]]*)?\\])?(?::([^}}]*))?}}', 'g');
+  new RegExp(
```

Comment:
something to do in a different PR but we should consider having a _client_ that encapsulates the use of the regular expression and return an object with the different components